docs/usage/setup-azure-devops.md

```diff
@@ -79,6 +79,39 @@ module.exports = {
 For the `endpoint` key, replace `YOUR-ORG` with your Azure DevOps organization.
 For the `repositories` key, replace `YOUR-PROJECT/YOUR-REPO` with your Azure DevOps project and repository.
 
+### Yarn users
+
+To do a succesfull `yarn install` you need to match the URL of the registry fully.
+Use the `baseUrl` config option to specify the full path to the registry.
+
+```javascript
+module.exports = {
+  platform: 'azure',
+  endpoint: 'https://<redacted>.visualstudio.com/',
```

Comment:
Do we want to keep using `<redacted>` in the Yarn configuration examples? The examples in the rest of the docs do not use `<redacted>`?

Would it be better to be consistent throughout maybe? Or is there a reason we use `<redacted>` in the Yarn configuration examples?

Reply:
maybe replace with something like `myorg` ?